embed/config.go

```diff
@@ -122,6 +122,12 @@ type Config struct {
 	// The map key is the route path for the handler, and
 	// you must ensure it can't be conflicted with etcd's.
 	UserHandlers map[string]http.Handler `json:"-"`
+
+	// auth
```

Comment:
newline here:
```
// auth

TokenType string
```

otherwise the `// auth` looks like a bad godoc for `TokenType` and CI will complain